zeppelin-zengine/src/main/java/org/apache/zeppelin/conf/ZeppelinConfiguration.java

```diff
@@ -314,6 +318,9 @@ public String getKeyManagerPassword() {
 
   public String getTrustStorePath() {
     String path = getString(ConfVars.ZEPPELIN_SSL_TRUSTSTORE_PATH);
+    if (path == null) {
+      path = getKeyStorePassword();
```

Comment:
Rhanks @FireArrow fixed getTrustStorePath, and getTrustStorePassword was already defaulting to getKeyStorePassword